guides/v2.1/extension-dev-guide/versioning/index.md

```diff
@@ -87,11 +85,10 @@ This example uses several composer packages on the public GitHub to simulate a m
     }
 }
 ```
```

Comment:
Indent code block 4 spaces (aligned with previous paragraph) so it doesn't break the numbering.